### ISSUE TITLE
Unique Visitors option should be available on Visitors Map for all date/period options.

### ISSUE THREAD

**Issue by guywillett** [matomo-org/matomo#15739]
Currently, viewing `Unique Visitors` on the `Visitors Map` is only possible when the "day" period is selected.
![Screen Shot 2020-03-27 at 07 46 29](https://user-images.githubusercontent.com/1755389/77729596-56185400-6fff-11ea-88ca-349de6d12eb9.png)
If any other period (in this example, a week)is applied then unique visits is no longer an option to be displayed:
![Screen Shot 2020-03-27 at 07 47 02](https://user-images.githubusercontent.com/1755389/77729627-70523200-6fff-11ea-876d-d17293729ad8.png)
I think this should at least be configurable, and probably respect the config as described here [https://matomo.org/faq/how-to/faq_113/](https://matomo.org/faq/how-to/faq_113/)
![Screen Shot 2020-03-27 at 07 53 07](https://user-images.githubusercontent.com/1755389/77729910-0be3a280-7000-11ea-9d37-226b590cedc0.png)

**Comment by tsteur** [comment]
Good point @guywillett

**Comment by tsteur** [comment]
Thanks for creating the issue 👍

**Comment by sgiehl** [comment]
I guess the reason is, that we are actually not calculating the exact unique visitors for the location reports. For periods bigger than day, there is only the sum of unique daily visitors, which is not the same. Not sure if we should show that as unique visitors 🤔

**Comment by guywillett** [comment]
> For periods bigger than day, there is only the sum of unique daily visitors, which is not the same

By this I assume that you mean the same "unique visitor" may visit on different days in the period and therefore is not "unique" for the period? In which case, I see what you mean - a bit tricky!

But is this not the same conundrum that has been solved and implemented in other parts of matomo that do show unique visits for different time periods? (and is configured by `enble_processing_unique_vistors_PERIOD`).

**Comment by sgiehl** [comment]
Right, we do that processing for some other reports. But that means we can't (only) simply combine the reports already generated for day period, but would need to completely process all raw log data for the bigger periods. Depending on the amount of data tracked, that can be quite a resource killer...

**Comment by guywillett** [comment]
BTW thanks for the attention and replies to this issue :-)

Yes, I understand that it can be quite a resource killer and may take a long time - but perhaps that is why it should be configurable and users opt-in to it.

**Comment by tofijak** [comment]
Any updates to this issue?
